packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -3,42 +3,39 @@ import filter from 'lodash/filter';
 import get from 'lodash/get';
 import map from 'lodash/map';
 import sortBy from 'lodash/sortBy';
+import animateScrollTo from 'animated-scroll-to';
 import { API_GUIDES } from '../../../project.constants';
 
-import {
-  NAME_PATTERN,
-  MAX_NAME_LENGTH,
-  MIN_NAME_LENGTH,
-} from './add.constants';
 import { ENGINES_STATUS } from '../../../../../components/project/storages/databases/engines.constants';
+import { ENGINE_LOGOS } from '../databases.constants';
+import { ORDER_KEYS, nameGenerator } from './add.constants';
 
 export default class {
   /* @ngInject */
   constructor(
-    $anchorScroll,
     $translate,
-    $q,
     CucCloudMessage,
     DatabaseService,
-    Poller,
+    ovhManagerRegionService,
+    $scope,
+    $timeout,
   ) {
-    this.$anchorScroll = $anchorScroll;
     this.$translate = $translate;
-    this.$q = $q;
     this.CucCloudMessage = CucCloudMessage;
     this.DatabaseService = DatabaseService;
-    this.Poller = Poller;
-    this.NAME_PATTERN = NAME_PATTERN;
-    this.MIN_NAME_LENGTH = MIN_NAME_LENGTH;
-    this.MAX_NAME_LENGTH = MAX_NAME_LENGTH;
+    this.ENGINE_LOGOS = ENGINE_LOGOS;
+    this.ovhManagerRegionService = ovhManagerRegionService;
+    this.orderKeys = ORDER_KEYS;
+    this.$timeout = $timeout;
```

Comment:
Move upper, related to this :
https://github.com/ovh/manager/pull/6905/files#r875864395